### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by TonySma**
## Description
Add french translation.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] I updated the documentation (see [here](https://github.com/kevinpapst/AdminLTEBundle/tree/master/Resources/docs))
- [x] I agree that this code is used in AdminLTEBundle and will be published under the [MIT license](https://github.com/kevinpapst/AdminLTEBundle/blob/master/LICENSE)

### CODE REVIEW

**Comment by kevinpapst** [comment]
Thanks @TonySma 👍 really appreciate your contribution!

File: Resources/translations/AdminLTEBundle.fr.xliff [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                <source>%progress%% Complete</source>
+                <target>%progress%% réussi</target>
+            </trans-unit>
+            <trans-unit id="This is a mandatory field">
+                <source>This is a mandatory field</source>
+                <target>Ce champ est obligatoire</target>
+            </trans-unit>
+        </body>
+    </file>
+</xliff>e
```
Comment (kevinpapst) [inline Resources/translations/AdminLTEBundle.fr.xliff]: Can you fix the file please

**Comment by kevinpapst** [comment]
Oh and you add french to the list of translations in the README file.

File: Resources/translations/AdminLTEBundle.fr.xliff (hunk 1 above)
Comment (TonySma) [reply Resources/translations/AdminLTEBundle.fr.xliff]: Done, sorry for this typo

**Comment by TonySma** [comment]
> Oh and you add french to the list of translations in the README file.

Done

**Comment by kevinpapst** [comment]
Awesome, thanks @TonySma 🎉

**Comment by kevinpapst** [comment]
and 2.8.4 released

**Pull request merged**